changelog.d/9053.misc

```diff
@@ -0,0 +1 @@
+Various clean-ups to the structured logging and logging context code.
```

Comment:
I kinda wonder if this is a different `bugfix`. It should mean that we correctly track some CPU usage that was being lost before.

Reply:
True, though tbh the measure block looks pretty pointless, its not doing very much in there at all

Reply:
doesn't it go off and wake up all the syncs and things?

Reply:
It just needs to look up each user and room in the dicts and call `.notify()`. I've updated the changelog anyway